manifests/init.pp

```diff
@@ -106,16 +110,18 @@
   String $varnish_storage_size         = '1G',
   Stdlib::Absolutepath $varnish_secret_file          = '/etc/varnish/secret',
   Stdlib::Absolutepath $varnish_storage_file         = '/var/lib/varnish-storage/varnish_storage.bin',
+  Optional[String[1]] $mse_config = undef,
+  Stdlib::Absolutepath $mse_config_file = '/etc/varnish/mse.conf',
   String $varnish_ttl                  = '120',
   Boolean $varnish_enterprise   = false,
   Boolean $varnish_enterprise_vmods_extra = false,
   Optional[Stdlib::Absolutepath] $vcl_dir                      = undef,
   Stdlib::Absolutepath $shmlog_dir                   = '/var/lib/varnish',
   Boolean $shmlog_tempfs                = true,
-  String $version                      = present,
+  String[1] $version                      = present,
   Boolean $add_repo             = false,
   Boolean $manage_firewall      = false,
-  String $varnish_conf_template        = 'varnish/varnish-conf.erb',
+  String[1] $varnish_conf_template        = 'varnish/varnish-conf.erb',
```

Comment:
nitpick: those changes are not related to the PR. Usually we try to split changes up into multiple PR so that all changes in one PR focus on one thing. I think in this case it's fine because the changes are quite small.